### ISSUE TITLE
Correct grammar on ogrlineref.rst

### ISSUE THREAD

**Issue by jidanni** [OSGeo/gdal#8465]
I got halfway through correcting the grammar on
https://github.com/OSGeo/gdal/blob/master/doc/source/programs/ogrlineref.rst
but then backed out... I fear any corrections I made to e.g.,
> The [:program:`ogrlineref`](https://github.com/OSGeo/gdal/blob/master/doc/source/programs/ogrlineref.rst#id4) creates a linear reference - a file containing a segments of special length (e.g. 1 km in reference units) and get coordinates, linear referenced distances or sublines (subpaths) from this file. The utility not required the M or Z values in geometry. The results can be stored in any OGR supported format. Also some information is written to the stdout.

would mess it up further. I think someone who understands the subject should step in here... Thanks.

**Comment by rouault** [comment]
I don't undestand what we are supposed to do with that ticket. You didn't even mention which corrections you did

**Comment by jidanni** [comment]
I'm sorry. I'm trying to say that I was going to correct that sentence, but I got halfway through it and then I chickened out. I think it needs some one who more understands what the original author was trying to say, in order to be able to correct the grammar. Thanks.

**Comment by rouault** [comment]
ah you mean correct "The utility not required the M or Z values in geometry" ?  I assume it should be "The utility does not require the M(easure) or Z components in the geometry"

**Comment by jidanni** [comment]
Good. Also maybe fix:

```
The :program:`ogrlineref` program can be used for:
                                              TO

create linear reference file from input data
       A

return the "linear referenced" distance for the projection of the input coordinates (point) on the path

return the coordinates (point) on the path according to the "linear referenced" distance
return the portion of the path according to the "linear referenced" begin and end distances
The :program:`ogrlineref` creates a linear reference - a file containing
a segments of special length (e.g. 1 km in reference units) and get
                                                                GETS

coordinates, linear referenced distances or sublines (subpaths) from
             AND

this file.
```

**Comment by rouault** [comment]
fixed per 2577691a8d
